.github/scripts/translation/make_translation.sh

```diff
@@ -0,0 +1,164 @@
+#!/bin/bash
+
+#
+# Check working directory
+#
+BASE_DIR="$(dirname "$0")"
+BASE_DIR="$( cd "$BASE_DIR" || exit 1; pwd )"
+export BASE_DIR
+
+#
+# Check external tools
+#
+PHP=$(which php)
+if [ -z $PHP ] ; then
+    echo -e "You must install php-cli before continue"
+    exit 1
+fi
+
+XGETTEXT=$(/usr/bin/which xgettext)
+if [ -z $XGETTEXT ] ; then
+    echo -e "You must install xgettext before continue"
+    exit 1
+fi
+
+MSGMERGE=$(/usr/bin/which msgmerge)
+if [ -z $MSGMERGE ] ; then
+    echo -e "You must install msgmerge before continue"
+    exit 1
+fi
+
+# Define projects
+PROJECTS=(
+    "centreon"
+)
+
+# Define language
+LANGS=(
+    "de_DE"
+    "es_ES"
+    "fr_FR"
+    "pt_BR"
+    "pt_PT"
+)
+
+#
+# Define global vars
+#
+PO_SRC=$BASE_DIR/po_src
+CAN_BE_TRANSLATE=false
```

Comment:
might be a nitpick but it should be renamed CAN_BE_TRANSLATED altogether

Reply:
fixed